src/components/widgets-settings/source-selector/source-selector.directive.coffee

```diff
@@ -0,0 +1,80 @@
+module = angular.module('impac.components.widgets-settings.source-selector',[])
+
+module.directive('settingSourceSelector', ($templateCache, $timeout, ImpacMainSvc, ImpacUtilities, ImpacTheming) ->
+  return {
+    restrict: 'A',
+    scope: {
+      parentWidget: '='
+      deferred: '='
+    },
+    template: $templateCache.get('widgets-settings/source-selector.tmpl.html'),
+
+    link: (scope) ->
+      w = scope.parentWidget
+
+      # Will there be a multi-org use case?
+      scope.mode ||= 'single'
```

Comment:
There is no binding for `mode`, so it will never be previously set, is this intentional is it's a todo?

Reply:
Yes, at the moment only single organization is being managed. I have added a `TODO` to make it more explicit